### DISCUSSION TITLE
Bounces to HTTP-endpoint are not being marked as "bounce"

### DISCUSSION THREAD

**Discussion by olegbliaher** [postalserver/postal#1536] [Help with using Postal]
Hi there, 

Using postal 1 here to send quite a lot of emails. I have also set up a route for the server's __returnpath__ that routes everything to an HTTP endpoint which is a script which simply logs all json as-is to a nice little (sometimes big) log.

I've just noticed that none of the messages in it are marked as "bounced", i.e "Undeliverable" automatic response that are otherwise marked as "bounced" on other servers are simply being logged as incoming mail. Furthermore, on other servers, with actual bounced emails, I can see a reference to the original email sent from our server to the remote one, as wekk as the bounced one. But on the one with the HTTP route I only see 

`Received a 200 from https://website.com/bounces/bounces.php`

Cold it be that it's skipping some check, to see if the incoming email is some kind of a response?

Any help would be very appreciated here!

**Comment by willpower232** [comment]
As per the documentation https://docs.postalserver.io/other/auto-responders-and-bounces bounces should be handled by Postal and other messages passed on to yourself.
